README.md

```diff
@@ -1,32 +1,65 @@
 # kat-client-proxies
-A library of client proxies for various apis
+A library of client proxies for various APIs for KAT.
 
-## Current proxies
-* myFTClient
-* acquisitionCtxClient
+KAT (Knowledge & administration tools) is a part of ft.com application created for Financial Times B2B clients.
 
-## Environment variables
-Runtime environment variables can be found in lib/config.js
-```
-MYFT_API_URL
-MYFT_API_KEY
-ACS_API_URL
-ACS_API_KEY
-USER_PROFILE_API_URL
-USER_PROFILE_API_KEY
-ALS_API_URL
-ALS_API_KEY
-FT_TOOL_ID
-FT_TOOL_ADMIN_ID
-FT_TOOL_DATE_ID
-```
-Test cases can also be configured via environment variables as per test/env.js
+## Getting started
+To get a copy of the project up and running on your local machine for development and testing purposes run `$ git clone git@github.com:Financial-Times/kat-client-proxies.git`.
+
+Use the `.env` file saved in LastPass to set up necessary environmental variables (feel free to approach KAT team via `#ft-syndikat` slack channel if you have any questions or access issues).
+
+Whenever you are adding a new client proxies, for maintenance don't forget to include it in the list in the end of this README.
 
 ## Testing
-`npm test` will by default use the mocked APIs and associated fixtures. It is possible to test against the actual APIs by setting the following environment variables:
+`$ npm test` will by default use the mocked APIs and associated fixtures. It is possible to test against the actual APIs by setting the following environment variables:
+
 ```
 USE_MOCK_API=false
 USER_UUID=
 LICENCE_UUID=
 ```
-Where `USER_UUID` is a valid user uuid on FT.com and `LICENCE_UUID` is the uuid of the licence associated with `USER_UUID`
+
+Where `USER_UUID` is a valid user uuid on FT.com and `LICENCE_UUID` is the uuid of the licence associated with `USER_UUID`. Feel free to approach KAT team via `#ft-syndikat` slack channel if you have any questions.
+
+## Deployment
+This component has been created to be included throughout other KAT components.
```

Comment:
Similar to what we recently spoke about, consider whether "module" would be more appropriate than "component"